bus/routes.go

```diff
@@ -1711,136 +1711,33 @@ func (b *Bus) handlePOSTAlertsRegister(jc jape.Context) {
 }
 
 func (b *Bus) accountsHandlerGET(jc jape.Context) {
-	jc.Encode(b.accountsMgr.Accounts())
-}
-
-func (b *Bus) accountHandlerGET(jc jape.Context) {
-	var id rhpv3.Account
-	if jc.DecodeParam("id", &id) != nil {
-		return
-	}
-	var req api.AccountHandlerPOST
-	if jc.Decode(&req) != nil {
-		return
-	}
-	acc, err := b.accountsMgr.Account(id, req.HostKey)
-	if jc.Check("failed to fetch account", err) != nil {
-		return
-	}
-	jc.Encode(acc)
-}
-
-func (b *Bus) accountsAddHandlerPOST(jc jape.Context) {
-	var id rhpv3.Account
-	if jc.DecodeParam("id", &id) != nil {
-		return
-	}
-	var req api.AccountsAddBalanceRequest
-	if jc.Decode(&req) != nil {
-		return
-	}
-	if id == (rhpv3.Account{}) {
-		jc.Error(errors.New("account id needs to be set"), http.StatusBadRequest)
-		return
-	}
-	if req.HostKey == (types.PublicKey{}) {
-		jc.Error(errors.New("host needs to be set"), http.StatusBadRequest)
-		return
-	}
-	b.accountsMgr.AddAmount(id, req.HostKey, req.Amount)
-}
-
-func (b *Bus) accountsResetDriftHandlerPOST(jc jape.Context) {
-	var id rhpv3.Account
-	if jc.DecodeParam("id", &id) != nil {
-		return
-	}
-	err := b.accountsMgr.ResetDrift(id)
-	if errors.Is(err, ibus.ErrAccountNotFound) {
-		jc.Error(err, http.StatusNotFound)
-		return
-	}
-	if jc.Check("failed to reset drift", err) != nil {
-		return
-	}
-}
-
-func (b *Bus) accountsUpdateHandlerPOST(jc jape.Context) {
-	var id rhpv3.Account
-	if jc.DecodeParam("id", &id) != nil {
+	var owner string
+	if jc.DecodeForm("owner", &owner) != nil {
 		return
-	}
-	var req api.AccountsUpdateBalanceRequest
-	if jc.Decode(&req) != nil {
+	} else if owner == "" {
+		jc.Error(errors.New("owner is required"), http.StatusBadRequest)
```

Comment:
Probably fine but at the same time it's a little awkward you can't fetch all the accounts from the `bus` seeing as there's no way to return all owners.